page-objects/pages-common.ts

```diff
@@ -1,3 +1,5 @@
 export const DEFAULT_CLICK_OPTIONS = { delay: 1000 };
 
-export const PK_WITH_COMPLETED_SAN = '787703'
\ No newline at end of file
+export const PK_WITH_COMPLETED_SAN = '787703'
+
+export const GOAL_CREATED_DATA = 'Plan created on 24 February 2025 by Dylan Hettinger'
```

Comment:
So presumably this test relies on some data already being in the database on the Test environment?

Reply:
Yes that's right, that PK has been setup for tests with an existing user.

Reply:
It would be nice to set up fixtures for this sort of stuff. I got really excited about running this against any env 😄  (dev, test or even dockerised envs)